sensor/common/sensor/sensor.go

```diff
@@ -220,6 +231,21 @@ func (s *Sensor) Start() {
 	go s.communicationWithCentral(&centralReachable)
 }
 
+// newScannerDefinitionsRoute returns a custom route that serves scanner
+// definitions retrieved from Central.
+func newScannerDefinitionsRoute(centralEndpoint string) (*routes.CustomRoute, error) {
+	handler, err := scannerdefinitions.NewDefinitionsHandler(centralEndpoint)
+	if err != nil {
+		return nil, err
+	}
+	return &routes.CustomRoute{
+		Route:         "/scanner/definitions",
+		Authorizer:    idcheck.ScannerOnly(),
+		Compression:   true,
```

Comment:
If we do end up compressing the files on Central's side, then no need to compress here